r1cs/gadgets/std/src/bits/boolean.rs

```diff
@@ -733,6 +784,39 @@ impl Boolean {
 
         Ok(current_run)
     }
+
+    /// Given a sequence `bits` of Booleans, constructs a `LinearCombination<ConstraintF>`
+    /// representing the field element whose little-endian bit representation corresponds to the
+    /// input `bits`. `one` represents the fixed variable of a constraint system employed to represent
+    /// constants in the linear combination. The function returns the constructed linear combination and
+    /// the field element represented by the linear combination, if all the `bits` in the sequence have values.
+    /// In addition, the function also returns a flag which specifies if there are no "real" variables
+    /// in the linear combination, that is if the sequence of Booleans comprises all constant values.
+    /// Assumes that `bits` can be packed in a single field element (i.e., bits.len() <= ConstraintF::Params::CAPACITY).
+    pub fn bits_to_linear_combination<'a, ConstraintF:Field, CS: ConstraintSystemAbstract<ConstraintF>>(_cs: CS, bits: impl Iterator<Item=&'a Boolean>) -> (LinearCombination<ConstraintF>, Option<ConstraintF>, bool)
```

Comment:
Split the definition of the function in multiple lines.

Reply:
Ok, I will also run `cargo fmt` to automatically fix these formatting issues